src/css/components/Modal.scss

```diff
@@ -0,0 +1,82 @@
+$Modal-bgColor: $cu-foreground !default;
+$Modal-overlayColor: $ts-black !default;
+$Modal-borderColor: $cu-background !default;
+$Modal-borderWidth: $su-small;
+$Modal-animation: 300ms cubic-bezier(0.2, 0.8, 0.4, 1.0);
+
+// * Modal Overlay
+// *
+// * 1. Position overlay so that it covers the viewport
+// * 2. Align modal content in the center of the viewport
+
+.Modal {
+  display: none;
+  position: fixed; // 1
+  top: 0px; // 1
+  bottom: 0px; // 1
+  left: 0px; // 1
+  right: 0px; // 1
+  background-color: rgba( $Modal-overlayColor, .5 );
+  z-index: 1000;
+  justify-content: center; // 2
+  align-items: center; // 2
+}
+
+.Modal.is-open {
+  display: flex; // 2
+}
+
+.Modal.is-closing {
+  animation: overlayOut $Modal-animation forwards;
+}
```

Comment:
Depending on how we handle the JS for this pattern, some of this may need to change. For now these states exist to show the modal and to animate its closing.